hieradata/clients/vpn.jenkins.io.yaml

```diff
@@ -9,3 +9,13 @@ profile::openvpn::networks:
+# DH comes from https://github.com/jenkins-infra/docker-openvpn
```

Comment:
```suggestion
# DH comes from https://github.com/jenkins-infra/docker-openvpn, it shouldn't have to be updated
```
Not sure if it should be added. WDYT?

Reply:
Disagree on this suggestion: the comment implies to never update DH while there are cases where it is required.